Makefile

```diff
@@ -50,6 +51,16 @@ bin:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/...
 
+trivy-bin:
+	curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b $@ v0.34.0
+
+check-%:
+	@: $(if $(value $*),,$(error $* is undefined))
+
+.PHONY: trivy-scan
+trivy-scan: trivy-bin check-TRIVY_TARGET_IMAGE
+	$(CURDIR)/trivy-bin/trivy image --exit-code 1 --severity CRITICAL,HIGH --ignore-unfixed $(TRIVY_TARGET_IMAGE)
```

Comment:
actually I forgot a few things:
- could you make `trivy-bin` a hidden folder
- could you delete the folder in the `clean` target
- could you add the folder to `.gitignore`

Reply:
thanks，I will

Reply:
done